_posts/2020-11-11-compexpo.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "LLNL's First Computing Virtual Expo"
+categories: event-report
+---
+
+The LLNL Computing Virtual Expo was an end-to-end digital experience with interactive booths, networking opportunities, and on-demand presentations, held on September 30. Lab employees and the public were invited to learn about new initiatives while networking and engaging with the Computing community, including computer scientists, IT experts, HPC contacts, and software developers and engineers.
+
+In addition to an exhibit booth about open source projects in general, the Expo featured booths for [RADIUSS project](https://software.llnl.gov/radiuss/) and the [Maestro Workflow Conductor](https://github.com/LLNL/maestrowf). [Ian Lee](https://github.com/IanLee1521) presented a lightning talk about the Lab's open source community and policies, and [Rob Neely](https://github.com/neely4) gave a talk on RADIUSS.
+
+Materials:
+- Releasing Open Source Software at the Lab ([poster](../assets/files/COMP_Poster_OSS.pdf))
+- Video: Open Source at LLNL (11:31) (*YouTube link forthcoming*) ([slides](../assets/files/2020CompExpo_Open_Source.pdf))
```

Comment:
I like this a lot. I think it's better to use `/assets/files/...` instead of `../assets/files/...` that way all the paths are relative to the root of the site (where they live) rather than from this posts directory though.